docs/HowTo/Configure/KeyManagement.md

```diff
@@ -0,0 +1,117 @@
+---
+Description: Configure key management
+---
+
+# Key management
+
+Configure ConsenSys Rollups operator keys to be managed in your local filesystem or by
+[Quorum Key Manager (QKM)](https://docs.quorum-key-manager.consensys.net/en/stable/).
+
+You can do this by updating the [engine configuration file](Configuration-File.md#engine-configuration-file).
+
+## Configure filesystem
+
+To configure the filesystem to manage Rollups operator keys, create a file containing the operator's private and
+encryption keys as in the following example:
+
+!!! example "`operator_1.acc`"
+
+    ```JSON
+    {
+        "account_key": {
+            "priv_key": "202454d1b4e72c41ebf58150030f649648d3cf5590297fb6718e27039ed9c86d"
+        },
+        "encryption_key": {
+            "key": "202454d1b4e72c41ebf58150030f649648d3cf5590297fb6718e27039ed9c86d"
+        }
+    }
+    ```
+
+Then, set the
+[key management configuration options](../../Reference/Configuration-File.md#key_managementaccount_key) in the engine
+configuration file as in the following example.
+
+!!! example "Filesystem key management configuration"
+
+    ```toml
+    [key_management.account_key]
+        manager_type = "Filesystem"
+
+        # Path to the file containing the operator's private key
+        operator_key_path = "node-data/test/keys/operator_1.acc"
+
+    [key_management.encryption_key]
```

Comment:
It's also worth noting that the `key_management.encryption_key` section can be omitted. This is mostly useful for a Consensus rollup